pages.nl/sunos/snoop.md

```diff
@@ -0,0 +1,25 @@
+# snoop
+
+> Network packet sniffer.
+> SunOS equivalent van tcpdump.
+> Meer informatie: <https://www.unix.com/man-page/sunos/1m/snoop>.
+
+- Bekijk de paketten van een specifieke netwerk interface:
+
+`snoop -d {{e1000g0}}`
+
+- Slaag de paketten op in een bestand, in plaats van ze weer te geven:
+
+`snoop -o {{filename}}`
+
+- Toon de verboze protocal layer samenvatting van de paketten in een bestand:
+
+`snoop -V -i {{filename}}`
```

Comment:
Please translate the placeholders, like `filename`, `hostname`, `port`, etc., for get this easier for non-English speakers.